_select.scss

```diff
@@ -0,0 +1,155 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+
+// Whether to draw the default down-arrow icon.
+$select__draw-caret: true !default;
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/triangle';
+@import 'level/level';
+
+
+// Select root
+// ---
+//
+// 1. Allow positioning of the native select element and caret.
+// 2. Match default user agent styling for inputs.
+
+.c-select {
+    position: relative; // 1
+
+    display: block;
+    max-width: 100%;
+    min-height: $tap-size;
+    background: #fff; // 2
+    border: 1px solid $border-color;
+
+    border-radius: $border-radius;
+
+    font-family: inherit;
+
+    user-select: none;
+
+
+    // Caret (down-arrow) icon
+    // ---
+    //
+    // 1. Position the caret in from the right edge.
+    // 2. Vertically center the caret.
+    // 3. Ensure caret does not block taps.
+
+    @if ($select__draw-caret) {
+        &::after {
+            @include triangle(
+                $type: 'triangle',
+                $direction: 'down',
+                $size: 7px,
+                $color: $border-color
+            );
+
+            content: '';
+
+            position: absolute; // 1
+            top: 0; // 1
+            right: $horizontal-input-padding; // 1
+            bottom: 0; // 1
+
+            margin: auto; // 2
+
+            pointer-events: none; // 3
+        }
+    }
+
+
+    // Native select element
+    // ---
+    //
+    // 1. Position the native select overtop of the custom element that
+    //    contains it.
+    // 2. Hide everything except the text value.
+
+    select {
+        position: absolute; // 1
+        top: 0; // 1
+        left: 0; // 1
+
+        width: 100%; // 1
+        height: 100%; // 1
+        padding-right: $horizontal-input-padding * 3.5;
+        padding-left: $horizontal-input-padding;
+        border: 0; // 2
+
+        background: none; // 2
+
+        -webkit-appearance: none; // 2
+    }
+}
+
+// Select with Inner Label
+// ---
+//
+// 1. Vertically align inner label and value.
+// 2. Hide the native select completely while it remains tappable.
+// 3. Avoid overlapping the caret.
+
+.c-select.c--inner-label {
+    display: flex;
+    align-items: center; // 1
+
+    select {
+        opacity: 0; // 2
+    }
+}
+
+
+// Inner label element
+// ---
+
+.c-select__label {
+    display: block;
+    margin: 0 1em 0 $unit;
```

Comment:
Hmm, well I guess in this case the label is visually "inside" the form input, giving the illusion that it's the input.

In this case, maybe it would make sense to use `$horizontal-input-padding`.

Let's keep this as food for thought. If we want to change this we can do so in a future update.